commons-executors/src/main/java/com/palantir/common/concurrent/PTExecutors.java

```diff
@@ -119,6 +119,12 @@ public static ExecutorService newCachedThreadPool(String name) {
         return newCachedThreadPoolWithMaxThreads(Short.MAX_VALUE, name);
     }
 
+    public static ExecutorService newCachedThreadPoolNoSpan(String name) {
```

Comment:
There were a substantial number of ways to call into newCachedThreadPool(String name).  I didn't think it was reasonable to change the behavior of all those paths in, so I created a new path that explicitly doesn't create a span.